infrastructure/main.tf

```diff
@@ -16,6 +16,11 @@ data "azurerm_key_vault_secret" "idam_oauth2_client_secret" {
   vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
 }
 
+data "azurerm_key_vault_secret" "postcode_lookup_token" {
+  name      = "postcode_lookup_token"
```

Comment:
notice name should use a hyphen rather  than underscore

Reply:
can you explain to me what is doing what, everywhere it's using underscore why here hyphen?

Reply:
no, we are not using underscore for the name. See other examples.